modules/accredited_representatives/app/controllers/accredited_representatives/application_controller.rb

```diff
@@ -10,20 +10,22 @@ class ApplicationController < ::ApplicationController
 
     before_action :verify_feature_enabled!
 
-    # TODO: Integrate SignIn Service to ARP Engine #76157
-    #   https://app.zenhub.com/workspaces/accredited-representative-facing-team-65453a97a9cc36069a2ad1d6/issues/gh/department-of-veterans-affairs/va.gov-team/76157
-    skip_before_action :authenticate
-
     private
 
+    def authenticate
+      # NOTE: this is currently a copy of app/controllers/concerns/authentication_and_sso_concerns.rb
+      # TODO: override the default authenticate method to use accredited representative authentication
+      if cookies[SignIn::Constants::Auth::ACCESS_TOKEN_COOKIE_NAME]
+        super
+      else
+        validate_session || render_unauthorized
+      end
+    end
+
```

Comment:
Could we delete this for now and let a future stretch of work (perhaps identity team controlled) address this functionality?